hoomd/hpmc/integrate.py

```diff
@@ -769,16 +812,16 @@ def __init__(self, seed, d=0.1, a=0.1, move_ratio=0.5,
 
         typeparam_shape = TypeParameter('shape', type_kind='particle_types',
                                         param_dict=TypeParameterDict(
-                                            vertices=list,
-                                            faces=list,
+                                            vertices=[(float, float, float)],
+                                            faces=[[int, int, int]],
```

Comment:
Ca
```suggestion
                                            faces=[[int, int, int]],  # TODO make faces use tuples instead of a list of exactly 3 elements.
```

Reply:
fixed.